server/GlideFormScratchpad.d.ts

```diff
@@ -1,3 +1,5 @@
-import { SNAPIGlideFormScratchpad } from "./SNAPIGlideFormScratchpad";
-declare class GlideFormScratchpad extends SNAPIGlideFormScratchpad {}
+import { SNAPIGlideFormScratchpad } from './SNAPIGlideFormScratchpad';
+declare class GlideFormScratchpad extends SNAPIGlideFormScratchpad {
+  [name: string]: any;
+}
```

Comment:
@bbarber9 this change is added due to need of using g_scratchpad in some BRs in our project :).

Reply:
I wonder if instead of creating this type we would be better off doing something like the following:
```typescript
declare let g_scratchpad:Partial<ShapeOfMyScratchPad>;
```
and then assigning it like that. That way we get type completions as well.
@knikolov-nuvolo @nabil-nuvolo  any thoughts on this?

Reply:
If we are OK with the idea I proposed, let's remove the line that turns this into a key-value object and we can resolve this 👍

Reply:
Done. You can check scratchpad implementation at https://github.com/nuvolo/core-apps/pull/217/commits/a0b5f5aa0891a1d563b9995f95d78f8bb47a9ded